plugins/inputs/windows_event_log/wineventlog/wineventlog.go

```diff
@@ -244,7 +233,7 @@ func (l *windowsEventLog) saveState(offset uint64) error {
 	return ioutil.WriteFile(l.stateFilePath, content, 0644)
 }
 
-func (l *windowsEventLog) read() ([]*windowsEventLogRecord, error) {
+func (l *windowsEventLog) read() []*windowsEventLogRecord {
```

Comment:
The `run()` function was just logging the error, nothing more.
If that is all that is going to be done, then this `read()` can log the error itself and not return it.
If anyone feels strongly, we can return the first error encountered or last error encountered (if any).
But we'd just be guessing about how someone might want to use this function in the future.

Reply:
I'm not opposed to the change.